File: pinyougou/pinyougou-order/pinyougou-order-interface/src/main/java/com/pinyougou/order/service/SeckillOrderService.java
package com.pinyougou.order.service;

import com.pinyougou.pojo.TbSeckillOrder;
import com.pinyougou.service.BaseService;
import com.pinyougou.vo.ChangeLong;
import com.pinyougou.vo.PageResult;

import java.util.List;

public interface SeckillOrderService extends BaseService<TbSeckillOrder> {

    PageResult search(Integer page, Integer rows, TbSeckillOrder tbSeckillOrder);

    /**
     * 查询全部订单并处理id过长
     * @return
     */
    List<ChangeLong> findAllSeckillOrder();

    /**
     * 查询全部订单并处理id过长(商家)
     * @param tbSeckillOrder
     * @return
     */
    List<ChangeLong> findAllSeckillOrderByUser(TbSeckillOrder tbSeckillOrder);


    /**
     * 状态查询
     * @param value
     * @return
     */
    List<ChangeLong> findStatus(String value);

}

File: pinyougou/pinyougou-manage-web/src/main/webapp/js/service/seckillorderService.js
app.service("seckillorderService", function ($http) {
    //加载列表数据
    this.findAll = function () {
        return $http.get("../seckillorder/findAll.do");
    };

    this.findPage = function (page, rows) {
        return $http.get("../seckillorder/findPage.do?page=" + page + "&rows=" + rows);
    };

    this.update = function (entity,id) {
        return $http.post("../seckillorder/update.do?id="+id, entity);
    };

    this.findOne = function (id) {
        return $http.get("../seckillorder/findOne.do?id=" + id);
    };

    this.search = function (page, rows, searchEntity) {
        return $http.post("../seckillorder/search.do?page=" + page + "&rows=" + rows, searchEntity);

    };

    this.findAllSeckillOrder = function () {
        return $http.post("../seckillorder/findAllSeckillOrder.do");
    };

    //状态查询
    this.findStatus = function (value) {
        return $http.get("../seckillorder/findStatus.do?value="+value);
    };
});